CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 # Changelog
+## [1.3.0] - 2021-03-24
+### Changes
```

Comment:
Instead of changes, use `Breaking changes` and `Bug fixes`. For the inclusive language each parameter, should have attached its own description and should be on `Breaking Changes`

Reply:
FYI: the previous changelog entry used "Changes" not "Breaking Changes" since the role does not really break things at this time.

Reply:
Right!! I almost wanted to comment "but the role doesn't break API, does it"?

Reply:
Well, IMO, deprecating something is a "breaking change" even if we are not dropping it in this version, it will be in the future.. But I am fine then to use "Changes" if the previous entry did.

Reply:
"deprecating" does not necessarily mean that it will be dropped in the future (https://en.wikipedia.org/wiki/Deprecation)

Taking deprecation as an excuse to break API later, is IMO bad (but common). We should not do that.

In any case, it would be a breaking change only in the future when it gets actually dropped.


Maybe one could say: but now I get a deprecation warning, and that already broke my setup. E.g. if libnm marks API as deprecated, you may get a compile time error with `-Wdeprecated-declarations=error`. Now one might argue whether that really counts as broken or still within acceptable limits. But if that is the concern, then the deprecation warning is too aggressive, and should be toned down. But not say: "well of course it broke, we depreacted it!".

Reply:
Using deprecation is not an excuse but a warning so that users can start to migrate away from it.